pkg/init/asker/asker_test.go

```diff
@@ -0,0 +1,85 @@
+package asker
+
+import (
+	"reflect"
+	"testing"
+)
+
+func Test_buildPersonalizedConfigurationOptions(t *testing.T) {
+	type args struct {
+		configuration ContainerConfiguration
+	}
+	tests := []struct {
+		name        string
+		args        args
+		wantOptions []string
+		wantTracker []OperationOnContainer
+	}{
+		{
+			name: "default options",
+			args: args{configuration: ContainerConfiguration{
+				Ports: []string{},
+				Envs:  map[string]string{},
+			}},
+			wantOptions: []string{"NOTHING - configuration is correct", "Add new port", "Add new environment variable"},
+			wantTracker: []OperationOnContainer{{
+				Ops:  "Nothing",
+				Kind: "",
+				Key:  "",
+			}, {
+				Ops:  "Add",
+				Kind: "Port",
+				Key:  "",
+			}, {
+				Ops:  "Add",
+				Kind: "EnvVar",
+				Key:  "",
+			}},
+		},
+		{
+			name: "all options",
+			args: args{configuration: ContainerConfiguration{
+				Ports: []string{"7000", "8000"},
+				Envs:  map[string]string{"env1": "env2"},
+			}},
+			wantOptions: []string{"NOTHING - configuration is correct", "Delete port \"7000\"", "Delete port \"8000\"", "Add new port", "Delete environment variable \"env1\"", "Add new environment variable"},
```

Comment:
Could you place strings in separates lines in wantOptions, so we don't need to scroll and we can see the parallel easily with the wantTracker content?